apps/total-typescript/src/video/lesson-list.tsx

```diff
@@ -246,18 +243,13 @@ const SolutionLink = ({
   path: string
 }) => {
   const router = useRouter()
-  const {data: solution} = useQuery(['solution', exercise.slug], async () => {
-    const fullExercise = await fetch(
-      `/api/skill/exercises/${exercise.slug}`,
-    ).then((res) => res.json())
-    return SolutionSchema.parse(fullExercise.solution)
-  })
```

Comment:
was migrating this to trpc and noticed it was only called here and didn't think it was needed based on the logic below so I removed it